lib/samlp.js

```diff
@@ -210,6 +210,10 @@ module.exports.parseRequest = function(req, options, callback) {
     var issuer = xpath.select("//*[local-name(.)='Issuer' and namespace-uri(.)='urn:oasis:names:tc:SAML:2.0:assertion']/text()", samlRequestDom);
     if (issuer && issuer.length > 0) data.issuer = issuer[0].textContent;
 
+
+    var subject = xpath.select("//*[local-name(.)='Subject']/*[local-name(.)='NameID']", samlRequestDom);
```

Comment:
Can you please also check  the namespace ?

Reply:
sure